src/components/Greetings/styles.ts

```diff
@@ -1,4 +1,5 @@
 import styled, { keyframes } from 'styled-components'
+import { ReactComponent as ReactLogo } from '../../assets/logo.svg'
```

Comment:
Can't we import it as default? 
```js
import Logo from '../../assets/logo.svg'
```
This example is using it with default exported: https://react-svgr.com/docs/webpack/